README.md

```diff
@@ -52,15 +57,15 @@ end
 
 You can then start synchronizing files with S3 through ```middleman s3_sync```.
 
-### Configuration Defaults
+#### Configuration Defaults
 
 The following defaults apply to the configuration items:
 
 | Setting                    | Default                            |
 | -----------------          | ----------------------------       |
-| aws_access_key_id          | ```ENV['AWS_ACCESS_KEY_ID']```     |
-| aws_secret_access_key      | ```ENV['AWS_SECRET_ACCESS_KEY']``` |
-| bucket                     | ```ENV['AWS_BUCKET']```            |
+| aws_access_key_id          | -                                  |
```

Comment:
S3 Sync (and fog) both support the use of environment variables to pass in the AWS credentials. They are effectively the default for s3_sync. We should continue to mention this in the README.